keyboards/mechmini/v2/keymaps/wsturgiss/keymap.c

```diff
@@ -0,0 +1,108 @@
+		KC_ESC,     KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_ENT,
+		KC_LSPO,    KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     TD(1),    TD(2),    TD(3),    KC_RSPC,
+		KC_LCTL,    TO(3),    KC_LALT,  KC_LGUI,  MO(1),        KC_SPC,         MO(2),    KC_VOLD,  KC_MPLY,  KC_VOLU,  KC_GRV),
+
+	[raise] = LAYOUT_2u_space_ortho(
+		_______,    KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_DEL,
+		_______,    _______,  _______,  _______,  _______,  _______,  KC_LEFT,  KC_DOWN,  KC_UP,    KC_RIGHT, KC_QUOT,  _______,
+		_______,    _______,  _______,  _______,  _______,  _______,  _______,  _______,  KC_LBRC,  KC_RBRC,  _______,  _______,  
+		_______,    _______,  _______,  _______,  _______,      _______,        _______,  KC_MPRV,  _______,  KC_MFFD,  _______),
+
+	[lower] = LAYOUT_2u_space_ortho(
+		_______,    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,
+		_______,    RGB_TOG,  RGB_MOD,  RGB_VAI,  RGB_VAD,  RGB_HUI,  RGB_HUD,  RGB_SAI,  RGB_SAD,  _______,  _______,  _______,
+		_______,    TBL_FLP, _______, _______,  _______,  _______,  _______,  _______,  _______,  KC_HOME,  KC_END,   _______,
+		_______,    _______,  _______,  _______,  _______,      _______,        _______,  _______,  KC_PGDN,  KC_PGUP,  _______),
+
+
+	[game] = LAYOUT_2u_space_ortho(
+		KC_TAB,     KC_Q,     KC_W,     KC_UP,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_BSPC,
+		KC_ESC,     KC_A,     KC_LEFT,  KC_DOWN,   KC_RIGHT, KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     TD(0),    KC_ENT,
+		KC_LSFT,    KC_Z,     KC_X,     KC_C,      KC_V,     KC_B,     KC_N,     KC_M,     TD(1),    TD(2),    TD(3),    KC_RSFT,
+		KC_LCTL,    TO(0),    KC_LALT,  KC_LGUI,   _______,        KC_SPC,       _______,  KC_VOLD,  KC_MPLY,  KC_VOLU,  KC_GRV),
+};
+
+//change colors and rgb modes on layer change
+uint32_t layer_state_set_user(uint32_t state) {
+    switch (biton32(state)) {
+    case raise:
+        rgblight_mode(1);
+        rgblight_setrgb_purple ();
+        break;
+    case lower:
+        rgblight_mode(1);
+        rgblight_setrgb(0x00, 0xa3, 0x0d);
+        break;
+    case game:
+        rgblight_mode(1);
+        rgblight_setrgb_red ();
+        break;
+    default: // for any other layers, or the default layer
+        rgblight_mode(5);
+        rgblight_setrgb(0xFF, 0xB6, 0x00);
+        break;
+    }
+  return state;
+};
+
+enum function_id {
+    SHIFT_ESC,
+};
+
+//I think this is to cancel space cadet stuff
+const uint16_t PROGMEM fn_actions[] = {
+  [0]  = ACTION_FUNCTION(SHIFT_ESC),
+};
+
+//Macros
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case TBL_FLP:
+      if (record->event.pressed) {
+        // when keycode tableflip is pressed
+        SEND_STRING("");
```

Comment:
If you enable Unicode, you can use this: 
```suggestion
              send_unicode_hex_string("0028 30CE 0CA0 75CA 0CA0 0029 30CE 5F61 253B 2501 253B");
```
This will output: (ノಠ痊ಠ)ノ彡┻━┻

Reply:
Thanks! I only got halfway through implementing this before getting distracted.